src/impl/tpls/KokkosBlas_Host_tpl.cpp

```diff
@@ -443,6 +465,15 @@ namespace KokkosBlas {
                      a, &lda,
                      b, &ldb);
     }
+    template<>
+    void 
+    HostBlas<float>::gesv(int n, int rhs,
```

Comment:
`xGESV` routines are in LAPACK, not the BLAS.  They should really go in a separate `HostLapack` struct.

Reply:
Thanks, @mhoemmen . See my comment below.